css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size;
+
+	.persona-avatar {
```

Comment:
I want to double check that we're able to put the avatar / circular image elsewhere if we need to. See @jim-benson' s PR.

Reply:
It can be, the markup will look like this:

```html
<div class="persona">
    <figure class="persona-avatar">
        <img src="~/src/scaffold/media/avatar.png" alt="username" />
    </figure>
</div>
```